src/test/resources/fr/xephi/authme/settings/config-old.yml

```diff
@@ -315,8 +315,6 @@ Security:
     console:
         # Remove spam console
         noConsoleSpam: true
-        # Replace passwords in the console when player type a command like /login
-        removePassword: true
```

Comment:
This should stay; it tests an old config.yml version

Reply:
Ok, reverted that commit.

Reply:
Cheers ;)